autotx/db.py

```diff
@@ -154,6 +156,7 @@ def get_all(self) -> list[models.Task]:
             tasks.append(
                 models.Task(
                     id=task_data["id"],
+                    app_user_id=task_data["app_user_id"],
```

Comment:
Why is this necessary? The apps know their user ids..

Reply:
i needed the `app_user_id` to [filter the tasks of the user](https://github.com/agentcoinorg/AutoTx/blob/2641ad970ba2ac1169d41dcd797ab8d0e4eb5d05/autotx/server.py#L312) - is there other way to achieve that and maybe I am missing it?

Reply:
You can add a new function to db to do that. It'll also be more performant instead of loading all of the tasks and filtering in memory. Also, it'll not expose the id to the app which I think this does.